### DISCUSSION TITLE
How should I use this file?

### DISCUSSION THREAD

**Discussion by Magnus111111** [humhue/DAOTools#7] [Q&A] [answered]
I don't know how to use python. Sorry to repeat a previously reported question.

**Comment by humhue** [comment, marked as answer]
You have to install it, and then you have to use a command-line interpreter (like cmd.exe for windows) to type the commands shown in the other discussion.

**Comment by Magnus111111** [comment]
Oh, pip3 install construct was a command that didn't work properly on Windows OS. It worked correctly when I typed py -m pip install construct
thank you for the reply.
